google/cloud/firestore_v1/base_document.py

```diff
@@ -188,12 +188,10 @@ def create(self, document_data: dict) -> NoReturn:
     def set(self, document_data: dict, merge: bool = False) -> NoReturn:
         raise NotImplementedError
 
-    def update(
-        self, field_updates: dict, option: _helpers.WriteOption = None
-    ) -> NoReturn:
+    def update(self, field_updates: dict, option=None) -> NoReturn:
         raise NotImplementedError
 
-    def delete(self, option: _helpers.WriteOption = None) -> NoReturn:
+    def delete(self, option=None) -> NoReturn:
```

Comment:
Likewise here?

Reply:
```suggestion
   def delete(self, option: _helpers.WriteOption = None) -> NoReturn:
```

Reply:
When run the unit test it throws an error like `AttributeError: module 'google.cloud.firestore_v1._helpers' has no attribute 'WriteOption'`

Reply:
Solved the error after removing import of `DocumentReference` from _helper.py file